.drone.yml

```diff
@@ -260,7 +235,118 @@ steps:
   name: mysql-integration-tests
 trigger:
   event:
-  - pull_request
+    include:
+    - pull_request
+  paths:
+    exclude:
+    - '**.md'
+    include:
+    - '**'
```

Comment:
Based on a quick look doesn't this go against each other? Should be enough with `exclude`, no?

Reply:
Nah, tried it and for some reason works only if you specify both, or maybe I am missing something else…